test/unit/controllers/Cid_EntryController.test.js

```diff
@@ -27,6 +35,12 @@ describe('INTEGRATION Cid_EntryController', () => {
       );
     });
     describe('good payload', () => {
+      before(() => {
```

Comment:
it'd be safer to put this and the restore into `beforeEach` so that you know when its been called in the `it` its the test you're referring to and its not leaked between tests